### PR TITLE
Chirayu

### PR DESCRIPTION
**Pull request by CHIRAYU09**

### CODE REVIEW

File: userController.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { user } from '../models';
+ import { downloadResource } from '../util';
+
+ const controller = {};
+
+ controller.download = async (req, res) => {
+  const fields = [
+    {
+      label: 'Student ID',
+      value: 'googleID'
```
Comment (hmtj99) [inline userController.js]: By Student ID we mean the email id of the student needs to be sent.

File: userController.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    {
+      label: 'Student Name',
+      value: 'name'
+    },
+    {
+     label: 'Attendance Percentage',
+      value: 'attendance_percentage'
+    }
+  ];
+  const data = await user.findAll();
```
Comment (hmtj99) [inline userController.js:21]: User data does not contain the attendance_percentage directly, You will need to calculate it.

File: userController.js (hunk 1 above)
Comment (CHIRAYU09) [reply userController.js]: Ok. I'll do the changes

Comment (CHIRAYU09) [reply userController.js]: Changes done

**Comment by hmtj99** [comment]
outdated PR with old file structure

**Pull request closed without merging**